vendor/knative.dev/pkg/webhook/stats_reporter.go

```diff
@@ -99,8 +99,9 @@ func (r *reporter) ReportRequest(req *admissionv1.AdmissionRequest, resp *admiss
 		return err
 	}
 
-	// TODO skonto: fix latency histogram
-	metrics.Record(ctx, requestCountM.M(1))
+	metrics.RecordBatch(ctx, requestCountM.M(1),
+		// Convert time.Duration in nanoseconds to milliseconds
+		responseTimeInMsecM.M(float64(d.Milliseconds())))
 	return nil
```

Comment:
This would be a regression of https://github.com/openshift/knative-eventing/pull/1301